linkinator.config.json

```diff
@@ -0,0 +1,10 @@
+{
+  "path": "/dist",
+  "recurse": true,
+  "skip": [
```

Comment:
There are some external links that fail, I'm not sure why they return non-success status codes 🤔 
This happens both locally and in the pipeline. It also happens when you use link checkers on our live site i.e.
```
npx linkinator https://docs.centrapay.com/
```

Reply:
Not sure if this is okay to leave or if it's a sign these links are wrong 👀